sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/ViewportPlugin.kt

```diff
@@ -0,0 +1,159 @@
+package com.mapbox.maps.plugin.viewport
+
+import com.mapbox.maps.plugin.MapPlugin
+import com.mapbox.maps.plugin.viewport.data.ViewportDataSource
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState.Following
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState.Idle
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState.Overview
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState.TransitionToFollowing
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraState.TransitionToOverview
+import com.mapbox.maps.plugin.viewport.state.ViewportCameraStateChangedObserver
+import com.mapbox.maps.plugin.viewport.transition.TransitionEndListener
+import com.mapbox.maps.plugin.viewport.transition.ViewportCameraTransition
+import com.mapbox.maps.plugin.viewport.transition.ViewportCameraTransitionOptions
+
+/**
+ * The Viewport plugin allows to track objects on a map.
+ *
+ * Most common use-case is the ability to track the user location and animate the location updates
+ * to create a navigation experience. Next to tracking a single object, the API allows tracking
+ * multiple objects. The viewport needs to automatically optimize the camera position to take in
```

Comment:
it's hard to say for me - is it true that we could already track multiple objects with given API?

Reply:
The documentation is from the SDD, we will continue working on these features in follow up PR, as this is just the initial PR that ports the nav camera and try to make it a working plugin.